### DISCUSSION TITLE
Pinchzoom not working if only 1 image

### DISCUSSION THREAD

**Discussion by 9dninedegrees** [nolimits4web/swiper#6089] [Q&A]
Hello,

We are using Swiper JS and are having some issues with the pinchzoom functionality on mobile.
If there is only 1 image in the slideshow this doesn't seem to work. On desktop (mobile emulated) I am able to double tap zoom, on mobile this doesn't work, pinchzooming (which I can't emulate) doesn't work on mobile either.

Slideshow with only 1 image: https://www.join.nl/stoelen/eetkamerstoelen/eetkamerstoel-lowen-bruin-boucle/
Slideshow with more then 1 image: https://www.join.nl/tafels/salontafels/salontafel-zion-zwart-mangohout-130x70cm/

Does anyone know what could be causing this or is this a bug in Swiper JS?

**Comment by zackrspv** [comment]
I'm partially able to replicate this, however, it does work EVENTUALLY, it's just super slow, and very inconsistent. For example, you can pinch/zoom on it, and have to do it like 5 times, and still it doesn't obey where it should be doing it. When it does, it finally moves the image, but keeps moving it up/down - even the page moves, like it's ignoring the scroll stuff. Frustrating. Still haven't found a resolution to how to do pinch/zoom on mobile with swiper -.-
